lib/fake-xhr/index.test.js

```diff
@@ -1923,7 +1925,9 @@ describe("FakeXMLHttpRequest", function () {
                 readyStateCb;
             var workingXHROverride = function () {
                 this.addEventListener = function (str, fn) {
-                    readyStateCb = fn;
+                    if (str === "readystatechange") {
+                        readyStateCb = fn;
+                    }
                 };
```

Comment:
this addEventListener is called with all the listeners in row and before the last listener was readyStateCb, but now is is the last, so we should filter